### DISCUSSION TITLE
Welcome to EX-WebThrottle2 Discussions!

### DISCUSSION THREAD

**Discussion by FrightRisk** [DCC-EX/EX-WebThrottle2#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

  ➡️ We can **convert issues to discussions** either individually or bulk by labels for issues labeled “question” or “discussion”.

**Comment by ManiAkasapu** [comment]
Hello folks..
Just checking... Any progress on this?
